src/app/auth/types.ts

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export interface AuthLayoutProps {
```

Comment:
Mantenha o uso do type pra ficar igual aos outros e se for só adicionar o Children, a tipagem do Component já tipa isso